integrated_channels/cornerstone/constants.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+Constants used by the Cornerstone integrated channel.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+CORNERSTONE_SUBJECTS_MAPPING = {
```

Comment:
I think we should move these inside `cornerstone/exporters/content_metadata.py` seems more appropriate place to have this constant.